src/persist-client/examples/inspect.rs

```diff
@@ -93,6 +96,50 @@ pub struct StateArgs {
     pub(crate) blob_uri: String,
 }
 
+/// Arguments for viewing a blob
+#[derive(Debug, Clone, clap::Parser)]
+pub struct BlobArgs {
+    /// Shard to view
+    #[clap(long)]
+    shard_id: String,
+
+    /// Blob key (without shard)
+    #[clap(long)]
+    key: String,
+
+    /// Blob to use
+    ///
+    /// When connecting to a deployed environment's blob, the necessary connection glue must be in
+    /// place. e.g. for S3, sign into SSO, set AWS_PROFILE and AWS_REGION appropriately, with a blob
+    /// URI scoped to the environment's bucket prefix.
+    #[clap(long)]
+    blob_uri: String,
+}
+
+/// Arguments for viewing contents of a batch part
+#[derive(Debug, Clone, clap::Parser)]
+pub struct BlobBatchPartArgs {
+    /// Shard to view
+    #[clap(long)]
+    shard_id: String,
+
+    /// Blob key (without shard)
+    #[clap(long)]
+    key: String,
+
+    /// Blob to use
+    ///
+    /// When connecting to a deployed environment's blob, the necessary connection glue must be in
+    /// place. e.g. for S3, sign into SSO, set AWS_PROFILE and AWS_REGION appropriately, with a blob
+    /// URI scoped to the environment's bucket prefix.
+    #[clap(long)]
+    blob_uri: String,
+
+    /// Number of updates to output. Default is unbounded.
+    #[clap(long, default_value = "18446744073709551615")]
```

Comment:
let's make the default bounded :)

Reply:
I'm partial to unbounded, similar to how a SQL query returns everything unless a limit is passed in. if we want it to be bounded by default, IMO we have a duty to structure the output to show the total count / maybe offset so a user immediately knows they're not seeing the full view, e.g.

```json
{
  updates: [...],
  count: 100,
}
```

it's not hard to do, but, I'm not sure we want to spend cycles on it

Reply:
ha, SQL as an example of good/desirable UX is not particularly convincing :P, but various unix cli tools also agree with the precedent. sure, let's start with unbounded and see where it goes

Reply:
> ha, SQL as an example of good/desirable UX is not particularly convincing :P

LOL fair point